src/platform/node/metrics.js

```diff
@@ -0,0 +1,216 @@
+'use strict'
+
+const v8 = require('v8')
+const log = require('../../log')
+
+const INTERVAL = 10 * 1000
+
+let nativeMetrics = null
+
+let interval
+let client
+let time
+let cpuUsage
+let counters
+
+reset()
+
+module.exports = function () {
+  return {
+    start: () => {
+      const StatsD = require('hot-shots')
+
+      try {
+        nativeMetrics = require('../../../build/Release/metrics')
+        nativeMetrics.start()
+      } catch (e) {
+        log.error('Unable to load native metrics module. Some metrics will not be available.')
+      }
+
+      client = new StatsD({
+        host: this._config.hostname,
+        port: 8125, // TODO: allow to configure this
+        prefix: 'datadog.tracer.runtime.',
```

Comment:
I think I'd prefer `datadog.tracer.node.` or `datadog.tracer.runtime.node.`.

By including `node` in the metric name it'll be more explicit and easier to differentiate when looking at a list of metrics, and if we happen to share a metric name with another runtime then people don't have to remember whether to query by `runtime:node` (also means we can drop `runtime` and `language` for now from default tags)

Reply:
Replaced with `runtime.node.` prefix as per conversation.